groups/instance/cloud-init/files/bootstrap_commands.yaml

```diff
@@ -1,4 +1,7 @@
 runcmd:
+  - /opt/jfrog/artifactory/var/etc/security/keys/trusted/get-ldap-certificate.sh
+  - chmod 0400 /opt/jfrog/artifactory/var/etc/security/keys/trusted/ldap-certificate.pem
+  - chown artifactory:artifactory /opt/jfrog/artifactory/var/etc/security/keys/trusted/ldap-certificate.pem
```

Comment:
These steps could be made a part of the `get-ldap-certificates.sh` script, so it's self-contained.

Reply:
Cheers Wayne. Implemented as required